source/screens/LoginScreen.tsx

```diff
@@ -329,22 +332,24 @@ function LoginScreen(props) {
                 <Label strong>PERSONNUMMER</Label>
                 <LoginInput
                   colorSchema="neutral"
-                  returnKeyType={null}
+                  returnKeyType={undefined}
                   placeholder="ååååmmddxxxx"
                   value={personalNumber}
                   onChangeText={handlePersonalNumber}
                   keyboardType="number-pad"
                   maxLength={12}
                   onSubmitEditing={() => handleLogin(true)}
                   center
+                  onBlur={() => ({})}
```

Comment:
wouldn't it be better to just make `onBlur` and `onMount` optional in the LoginInput component?

Reply:
Yes definitely, however I wanted to avoid editing too many files unrelated to this PR, but now that I think about it this change is counter-productive instead. It might be better to revert this change and set `onBlur`/`onMount` to be optional outside of this PR.